person.rb

```diff
@@ -0,0 +1,28 @@
+# Person class
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1...100_000)
+    @name = name
+    @parent_permission = parent_permission
+    @age = age
+  end
+
+  private
+
+  def of_age?
+    return true if @age >= 18
+
+    false
+  end
+
+  public
+
+  def can_use_services?
+    return true if is_of_age? || @parent_permission
+
+    false
+  end
```

Comment:
- [ ] It would be better professionally if you move the private methods like `of_age?` in this case to the end of `person` class and move `can_use_services?` method just above it without the word `public`.
[Hint] kindly make it like that:
```ruby
  def can_use_services?
     is_of_age? || @parent_permission
  end

private

  def of_age?
      @age >= 18
  end
```

Reply:
Thanks for your suggestion. Fixed.

Reply:
Well resolved 👍🏽